django_test_migrations/signals.py

```diff
@@ -0,0 +1,13 @@
+from contextlib import contextmanager
+
+from django.db.models.signals import post_migrate, pre_migrate
+
+
+@contextmanager
+def mute_migrate_signals():
+    pre_migrate_receivers = pre_migrate.receivers
+    post_migrate_receivers = post_migrate.receivers
+    pre_migrate.receivers = post_migrate.receivers = []
```

Comment:
Following assignment will store the same `list` (because `list` is mutable) for both attributes which can lead to many hard do debug errors.

```suggestion
    pre_migrate.receivers = []
    post_migrate.receivers = []
```

Reply:
Great catch, thanks!